Gemfile.lock

```diff
@@ -403,6 +403,8 @@ GEM
     sqlite3 (1.3.11)
     sse-rails-engine (1.4.0)
       rails (~> 4.0)
+    sucker_punch (2.0.1)
```

Comment:
These jobs aren't critical so could be freely killed.  Worse case is if a server reboot happens, the user can request another export to run.  After it ages for a while, the next time a report is generated, it would be cleaned up.  Alternatively, if there is a place where something is run upon start, the exports with a pending or started status could be queued again.